e2e/pages/RegistrationDetails/RegistrationDetailsPage.ts

```diff
@@ -22,6 +22,11 @@ class RegistrationDetails {
   financialServiceProviderDropdown = 'app-update-fsp #select-label';
   debitCardPaTable = 'ion-card-title';
   debitCardStatus = 'ion-label';
+  tabButton = 'ion-button';
+  historyTile = 'article';
+  historyTileText = 'span';
+  historyTileTimeStamp = 'time';
+  tileInformationPlaceHolder = 'strong';
```

Comment:
Still not a fan of the very generic selectors we are using :D

Reply:
I'll definitely drop this ones as they make no sense anymore and probably will cause more harm in the future